repositories.yaml

```
@@ -567,3 +567,24 @@ applications:
         app_id: rally.debug
     encryption:
       use_jwk: true
+
+  - app_name: rally_core
+    canonical_app_name: Rally Core Add-on
+    app_description: |
+      The Rally Core Add-on orchestrates the installation
+      and the lifecycle of Rally studies.
```

Feels like it would be good idea to provide a link to Rally here, in case someone is looking at the Glean Dictionary and wondering what this is.

```suggestion
      and lifecycle of [Rally](https://rally.mozilla.org/) studies.
```